src/lamplib/src/genny/tasks/auto_tasks.py

```diff
@@ -169,10 +155,18 @@ class Workload:
     auto_run_info: Optional[List[AutoRunBlock]] = None
     """The list of `When/ThenRun` blocks, if present"""
 
-    def __init__(self, workspace_root: str, file_path: str, is_modified: bool, reader: YamlReader):
+    def __init__(
+        self,
+        workspace_root: str,
+        file_path: str,
+        is_modified: bool,
+        reader: YamlReader,
+        dry_run: bool,
+    ):
         self.workspace_root = workspace_root
         self.file_path = file_path
         self.is_modified = is_modified
+        self.dry_run = dry_run
```

Comment:
Is this used anywhere?

Reply:
I initially used it inside `Workload.variant_tasks()`, but then realized that mocking `expansions` is a better way to achieve the same thing (exercising the variant comparison logic). I've removed it from this class.